src/phoenix/config.py

```diff
@@ -74,9 +74,18 @@
 ENV_PHOENIX_SECRET = "PHOENIX_SECRET"
 ENV_PHOENIX_API_KEY = "PHOENIX_API_KEY"
 ENV_PHOENIX_USE_SECURE_COOKIES = "PHOENIX_USE_SECURE_COOKIES"
+ENV_PHOENIX_PASSWORD_RESET_TOKEN_EXPIRY = "PHOENIX_PASSWORD_RESET_TOKEN_EXPIRY"
 ENV_PHOENIX_ACCESS_TOKEN_EXPIRY = "PHOENIX_ACCESS_TOKEN_EXPIRY"
 ENV_PHOENIX_REFRESH_TOKEN_EXPIRY = "PHOENIX_REFRESH_TOKEN_EXPIRY"
 
+ENV_PHOENIX_SMTP_ENABLED = "PHOENIX_SMTP_ENABLED"
+ENV_PHOENIX_SMTP_USERNAME = "PHOENIX_SMTP_USERNAME"
+ENV_PHOENIX_SMTP_PASSWORD = "PHOENIX_SMTP_PASSWORD"
+ENV_PHOENIX_SMTP_MAIL_FROM = "PHOENIX_SMTP_MAIL_FROM"
+ENV_PHOENIX_SMTP_HOSTNAME = "PHOENIX_SMTP_HOST"
+ENV_PHOENIX_SMTP_PORT = "PHOENIX_SMTP_PORT"
```

Comment:
Why split these in two? One env var is better 
<img width="834" alt="Screenshot 2024-09-17 at 8 48 43 AM" src="https://github.com/user-attachments/assets/1f25d139-400b-472d-9246-139b0b3a3653">

Reply:
OK

Reply:
Actually, since the package we're using can only configure them [separately](https://github.com/sabuhish/fastapi-mail/blob/a4aff1acd9ba98ef84ada493b922aad3ae95115d/fastapi_mail/config.py#L12-L13), it's not worthwhile on our end if we have to spend the extra effort to parse them out of the URL.